charts/argo-cd/values.yaml

```diff
@@ -3019,6 +3019,12 @@ notifications:
     # -- Whether helm chart creates notifications controller secret
     create: true
 
+    # -- notifications controller Secret name
+    name: "argocd-notifications-secret"
+
+    # -- Whether to use existing secret. If true, assumes that the existing secret name is notifications.secret.name
+    useExistingSecret: false
```

Comment:
Do we really need an additional bool for that. I assume we can add a note on the existing bool `create`.

Reply:
I agree that having one would be best. 

The reason I added a boolean is the [conditional rule ](https://github.com/argoproj/argo-helm/blob/fc620a3d51afb448fd3a14ed86a8949ed3e47af6/charts/argo-cd/templates/argocd-notifications/clusterrole.yaml#L41) in the ClusterRole. I didn't know the code well enough when I first addressed it to understand why it's there so I didn't want to touch it.  

However, now that we are passing `- --secret-name={{ .Values.notifications.secret.name }}` I guess we can remove the condition completely, and always set the rule :
```
  - apiGroups:
    - ""
    resourceNames:
    - {{ .Values.notifications.secret.name }}
    resources:
    - secrets
    verbs:
    - get
```

Do you agree?

Reply:
Yes I think we can remove that conditional 👍